src/components/cookies/CookieConsent.tsx

```diff
@@ -25,20 +24,23 @@ export default function CookieBanner() {
 
   const theme = useTheme();
 
-  const handleOpen = () => {
-    setCookieConfigOpen(true);
-  };
-
   const { breakpoints } = useTheme();
   const isMobile = useMediaQuery(breakpoints.down('sm'));
 
+  const hasUserMadeChoice =
+    typeof window !== 'undefined' && localStorage.getItem('userAcceptedAnalytics') !== null;
+
+  // Note: If they have already chosen don't show again unless configured from footer
+  if (hasUserMadeChoice) return null;
+
   return (
     <>
-      <CookieConfigModal />
+      {/* <CookieConfigModal /> */}
```

Comment:
I think this component is dead now and can get removed

Reply:
removed `bbdab4e39cfe9181c05300f77fc342581dca429f`